File: Data Structures & Algorithms/majority-element/submission-2.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int cnt = 1;
        int candidate = nums[0];
        for (int i = 1;i<nums.size();i++){
            if (candidate==nums[i]){
	            cnt++;
	        }
	        else{
	            cnt--;
	        }
	        if (cnt==0){
	            candidate = nums[i];
                cnt = 1;
	        }
        }
        return candidate;
    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-2.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int len = 2 * nums.length;
        int arr[] = new int[len];
        for (int i = 0;i<nums.length;i++){
            arr[i] = nums[i];
        }
        for (int i = nums.length;i<len;i++){
            arr[i] = nums[i%nums.length];
        }
        return arr;

    }
}